### PR TITLE
Parent-child component event communication

### PR DESCRIPTION
**Pull request by PavlosIsaris**
Added parent-child component event communication example
Removed trailing '<' from component's featureDescription attribute.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: client/src/pages/guide/english/vue/components/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ can just reference it by using as a tag. In our example, we can use the tag
     iconSrc="https://freedesignfile.com/upload/2017/08/rocket-icon-vector.png"
     iconAltText="rocket"
     featureTitle="Processing speed"
-    featureDescription="Our solution has astonishing benchmarks grades">
+    featureDescription="Our solution has astonishing benchmarks grades"
```
Comment (lynxlynxlynx) [inline client/src/pages/guide/english/vue/components/index.md]: Now this tag is not complete any more.

File: client/src/pages/guide/english/vue/components/index.md [hunk 2]
```diff
@@ -66,3 +66,68 @@ as we setted `iconSrc` or `featureTitle` as they were valid attributes. And the
...
+      </h4>
+      <p class="description">
+        {{ featureDescription }}
+      </p>
+    </div>
+  `
+});
+```
+
+Now, when we create a `<feature-card>`, we can define a custom mehthod name to be called when `cardClicked` is fired:
```
Comment (lynxlynxlynx) [inline client/src/pages/guide/english/vue/components/index.md]: mehthod

File: client/src/pages/guide/english/vue/components/index.md [hunk 3]
```diff
@@ -66,3 +66,68 @@ as we setted `iconSrc` or `featureTitle` as they were valid attributes. And the
...
+  <feature-card
+    iconSrc="https://freedesignfile.com/upload/2017/08/rocket-icon-vector.png"
+    iconAltText="rocket"
+    featureTitle="Processing speed"
+    featureDescription="Our solution has astonishing benchmarks grades"
+    v-on:click="featureCardClicked"
+  </feature-card>
+</div>
+```
+Now, `featureCardClicked` needs to be a method in the parent component, and it's parameter will hold the data emmited from the `<feature-card>` component.
```
Comment (lynxlynxlynx) [inline client/src/pages/guide/english/vue/components/index.md]: emited

**Comment by moT01** [comment]
Thank you for your contribution, but the guide has been moved here https://github.com/freeCodeCamp/freeCodeCamp/tree/master/guide and this cannot be merged. If you create a new PR targeting the new location we will do our best to get it merged.

**Pull request closed without merging**